lib/record_store/provider/ns1.rb

```diff
@@ -1,10 +1,12 @@
 require_relative 'ns1/client'
+require 'net/http'
 
 module RecordStore
   class Provider::NS1 < Provider
     class Error < StandardError; end
 
     class << self
+      include Limiter
```

Comment:
Does not seem needed anymore

Reply:
Oops, nice catch! interesting that Rubocop didn't catch it.